build-system-tests/templates/template-react-native-aws-exports.js

```diff
@@ -0,0 +1,3 @@
+const awsmobile = {};
```

Comment:
With an empty aws-exports config, `auth` throw an error message with 'AuthError -' in console or terminal. With this error, we can tell if _react-native-log-ios_ and _log-android_ log errors for React Native CLI properly.